contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1016,7 +1016,7 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 						}
 					}
 				}
-				if (id->keyword()->TRIM() || id->keyword()->REPLACE() || id->keyword()->TRANSLATE() || id->keyword()->SUBSTRING() || id->keyword()->STRING_AGG()  || id->keyword()->CONCAT() || id->keyword()->CONCAT_WS())
+				if (id->keyword()->TRIM() || id->keyword()->REPLACE() || id->keyword()->TRANSLATE() || id->keyword()->SUBSTRING() || id->keyword()->STRING_AGG())
```

Comment:
Removed all the other functions, except `TRIM` since if we did not append sys to `TRIM()` function then it will always hit PostgreSQL `btrim` function from the grammer.